vampire_survivors/src/boss.py

```
@@ -0,0 +1,10 @@
+import pygame
+from enemy import Enemy
+
+class Boss(Enemy):
+    def __init__(self, pos, groups, obstacle_sprites, player):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)